src/layers/legacy/medCoreLegacy/database/medDatabaseExporter.cpp

```diff
@@ -11,28 +11,40 @@
 
 =========================================================================*/
 
-#include <dtkCoreSupport/dtkAbstractDataWriter.h>
 
 #include <medAbstractData.h>
 #include <medAbstractDataFactory.h>
+#include <medAbstractDataWriter.h>
 #include <medDatabaseExporter.h>
 
 class medDatabaseExporterPrivate
 {
 public:
     medAbstractData *data;
+    QList<medAbstractData*> dataList;
     QString          filename;
     QString          writer;
+    bool saveMultipleData;
 };
 
 medDatabaseExporter::medDatabaseExporter(medAbstractData * data, const QString & filename, const QString & writer) : medJobItemL(), d(new medDatabaseExporterPrivate)
 {
     d->data     = data;
     d->filename = filename;
     d->writer   = writer;
+    d->saveMultipleData = false;
 }
 
-medDatabaseExporter::~medDatabaseExporter()
+medDatabaseExporter::medDatabaseExporter(QList<medAbstractData*> data, const QString & filename, const QString & writer) : medJobItemL(), d(new medDatabaseExporterPrivate)
+{
+    d->data     = NULL;
```

Comment:
nullptr

Reply:
This is just a cherry-pick of previous code. I think we should stop making changes to existing code unless we are functionally changing that code. The recent problems with merging were caused in large part by that. This applies to things like changing NULL, indentation, parenthesis etc. I really think we should make it a habit to focus on functional changes. This does not mean we shouldn't clean the new code of course.

Reply:
Indeed I hesitated about these ones, however, if it's located in medInria, we won't have the problem i think

Reply:
We must do cosmetic changes later, I think